docs/getting-started.md

```diff
@@ -7,11 +7,11 @@ GraphQL Kotlin consists of number of libraries that aim to simplify running a Gr
 
 ## Modules
 
-* [graphql-kotlin-schema-generator](https://github.com/ExpediaGroup/graphql-kotlin/tree/master/graphql-kotlin-schema-generator)
-* [graphql-kotlin-federation](https://github.com/ExpediaGroup/graphql-kotlin/tree/master/graphql-kotlin-federation)
+* [graphql-kotlin-schema-generator](https://github.com/ExpediaGroup/graphql-kotlin/tree/master/graphql-kotlin-schema-generator)  ([Javadoc](https://www.javadoc.io/doc/com.expediagroup/graphql-kotlin-schema-generator))
+* [graphql-kotlin-federation](https://github.com/ExpediaGroup/graphql-kotlin/tree/master/graphql-kotlin-federation) ([Javadoc](https://www.javadoc.io/doc/com.expediagroup/graphql-kotlin-federation))
   &mdash; Schema generator extension to build federated GraphQL schemas
   &mdash; Code only GraphQL schema generation for Kotlin
```

Comment:
It looks like this line got moved down. Can you move this `&mdash; Code only GraphQL schema generation for Kotlin` to be inline with `graphql-kotlin-schema-generator`

Reply:
I've committed a fix for that.